CMakeLists.txt

```diff
@@ -157,17 +238,17 @@ if(WW3)
 
   add_custom_target(ww3_nems
     WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/WW3/model/esmf
-    COMMAND ${CMAKE_BUILD_TOOL} WW3_PARCOMPN=4 WW3_COMP=${WW3_COMP} ww3_nemslibonly > make.log 2>&1
+    COMMAND ${CMAKE_BUILD_TOOL} WW3_PARCOMPN=4 WW3_COMP=${WW3_COMP} ww3_nemslibonly > ${PROJECT_BINARY_DIR}/ww3_make.log 2>&1
   )
 
     set(WW3_LIBS ${PROJECT_SOURCE_DIR}/WW3/model/obj/libww3_multi_esmf.a)
-endif(WW3)
+endif()
 
 ###############################################################################
-### NEMS
+### NEMS (ufs-weather-model-lib)
 ###############################################################################
 add_library(
-    ufs-s2s-model-lib
+    ufs-weather-model-lib
```

Comment:
this is a s2s model, not weather model

Reply:
This will be addressed in my fixes i push today.. I had caught that this morning.